scripts/s3-sanity/s3-sanity-test.sh

```diff
@@ -226,6 +245,8 @@ echo "Create Account"
 output=$(s3iamcli createaccount -n SanityAccountToDeleteAfterUse  -e SanityAccountToDeleteAfterUse@sanitybucket.com --ldapuser sgiamadmin --ldappasswd "$ldappasswd" --no-ssl)
 access_key=$(echo -e "$output" | tr ',' '\n' | grep "AccessKeyId" | awk '{print $3}')
 secret_key=$(echo -e "$output" | tr ',' '\n' | grep "SecretKey" | awk '{print $3}')
+echo "access key " $access_key
```

Comment:
why we are printing credentials?

Reply:
that was a part of debugging. Removing now..